Tests/SwiftStdlibTests/SequenceExtensionsTests.swift

```diff
@@ -100,6 +100,11 @@ final class SequenceExtensionsTests: XCTestCase {
         XCTAssertFalse(["a", "b", "c", "d"].containsDuplicates())
     }
 
+    func testDuplicates() {
+        XCTAssertEqual([1, 1, 2, 2, 3, 3, 3, 4, 5].duplicates(), [1, 2, 3])
```

Comment:
Since the return result is unsorted, maybe check capture both results in a `Set`. E.g.
```Swift
XCTAssertEqual(Set([1, 1, 2, 2, 3, 3, 3, 4, 5].duplicates()), Set([1, 2, 3]))
```

Reply:
Hello! I sorted the array now.

Reply:
I saw, that was an even better solution!